services/collaboration/pkg/config/defaults/defaultconfig.go

```diff
@@ -32,6 +34,13 @@ func DefaultConfig() *config.Config {
 				Duration: "12h",
 			},
 		},
+		Store: config.Store{
+			Store:    "nats-js-kv",
+			Nodes:    []string{"127.0.0.1:9233"},
+			Database: "collaboration",
+			Table:    "",
+			TTL:      30 * time.Minute,
```

Comment:
@jvillafanez How does the WebOffice app refreshes the token?
I am aware of the 30 min, but i never saw a Web Office actually refresh the WOPI token.

Reply:
I don't see any info on how the refresh in supposed to work. What I found is in https://learn.microsoft.com/en-us/microsoft-365/cloud-storage-partner-program/rest/concepts#the-access_token_ttl-property

> To prevent data loss for users, Office for the web prompts users to save and refresh their sessions if the access token for their session is close to expiring. In order to do this, Office for the web needs to know when the access token expires, which it determines based on the access_token_ttl value.

So I guess you're expected to close the editor and reopen it (with a fresh token)